partcad/src/partcad/runtime_python.py

```
@@ -235,36 +235,7 @@ def run_onced_locked(self, cmd, stdin="", cwd=None, session=None, path=None):
-        #     pc_logging.debug("Output of %s: %s" % (cmd, stdout))
-        if stderr:
-            pc_logging.debug("Error in %s: %s" % (cmd, stderr))
-
-        # TODO(clairbee): remove the below when a better troubleshooting mechanism is introduced
-        # f = open("/tmp/log", "w")
-        # f.write("Completed: %s\n" % cmd)
-        # f.write(" stdin: %s\n" % stdin)
-        # f.write(" stderr: %s\n" % stderr)
-        # f.write(" stdout: %s\n" % stdout)
-        # f.close()
+        stdout, stderr = supera().run(cmd, stdin=stdin, cwd=cwd)
```

Ruff (0.8.2)

Undefined name `supera`

(F821)